packages/appstore/src/constants/trading-hub-content.tsx

```diff
@@ -13,39 +14,67 @@ type TTradingHubContents = Record<'step1' | 'step2' | 'step3' | 'step4' | 'step5
 
 export const trading_hub_contents: TTradingHubContents = {
     step1: {
-        component: 'Step1',
+        component: <StaticDashboard />,
         footer_header: localize('Welcome to Trading Hub'),
         footer_text: localize('This is your new dashboard'),
         has_next_content: false,
     },
     step2: {
-        component: 'Step2',
+        component: <StaticDashboard is_cfd_text_blurry is_options_text_blurry is_platformlauncher_blurry />,
         footer_header: localize('Trading accounts'),
         footer_text: localize(
             'These are the trading accounts available to you. You can click on an account to find out more.'
         ),
         has_next_content: false,
     },
     step3: {
-        component: 'Step3',
+        component: <StaticDashboard is_cfd_item_blurry is_options_item_blurry is_platformlauncher_blurry />,
```

Comment:
1.just CFD and Options title should be highlighted
2. no interaction should be available (buttons should not be clickable)
test link:
![image](https://user-images.githubusercontent.com/82078941/192281299-d9bcd55b-17c9-408f-9e0c-9c4b4d9580cf.png)

design:
![image](https://user-images.githubusercontent.com/82078941/192281331-750c0689-8b6b-4187-b55e-671ffe9952e1.png)